docs/networking/README.md

```diff
@@ -0,0 +1,7 @@
+
+# Rancher Desktop Network Documentation
+
+The table of contents below provides references to all the projects that comprise the Rancher Desktop network.
```

Comment:
This is all for Windows? We should note that then (and probably in the individual files).

Reply:
Should I create the following path? `/docs/networking/windows`

Reply:
You can if you want, but as long as the docs themselves make it clear it should be fine.